### PR TITLE
Add Privacy Policy and Honor DNT Headers (fixes #94 & #276)

### LINKED ISSUE #94: Privacy Policy [$15]
Lewis mentioned that it would be good to have a Privacy policy since we run Google analytics

<bountysource-plugin>

---
There is a **[<total-bounty>$15</total-bounty> open bounty](https://www.bountysource.com/issues/7707271-privacy-policy?utm_campaign=plugin&utm_content=tracker%2F10548672&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F10548672&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by lewisgoddard**
http://beta.elementaryos.org/branch/privacy/privacy-policy

Fixes #94 & #276.

### CODE REVIEW

**Comment by lewisgoddard** [comment]
I suppose we should have a link from every page, international standard and all that.

**Comment by KeitIG** [comment]
You just forgot a link to the new Privacy Policy page ;) (in footer I guess). Otherwise it's great.

**Comment by davidak** [comment]
DNT works, thank you :)

**Comment by cassidyjames** [comment]
@lewisgoddard some of the wording is messy. I'll propose changes here as I see 'em.

File: privacy-policy.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+    include '_templates/sitewide.php';
+    $page['title'] = 'Privacy Policy';
+    include '_templates/header.php';
+?>
+            <div class="row" style="max-width:1000px;">
+                <h1>Privacy Policy</h1>
+                <h2>Do Not Track</h2>
+                <p>This site honors the do-not-track setting found in most modern browsers, and disables Google Universal Analytics when it is present. Unfortunately, we cannot remove other tracking methods without severely impacting the use of the site. Should you wish to do so, reference the section on disabling cookies.</p>
```
Comment (cassidyjames) [inline privacy-policy.php]: > Unfortunately, we cannot remove other tracking methods without severely impacting the use of the site. Should you wish to do so, reference the section on disabling cookies.

Change to:

> We cannot automatically remove other tracking methods without severely impacting the use of the site. Should you wish to manually manage these tracking methods, reference the section on disabling cookies.

File: privacy-policy.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                <p>Below you will find links to enabling DNT in your browser.</p>
+                <ul>
+                    <li><a target="_blank" href="https://support.google.com/chrome/answer/114836">Google Chrome</a></li>
+                    <li><a target="_blank" href="https://www.mozilla.org/en-US/firefox/dnt/">Mozilla Firefox</a></li>
+                    <li><a target="_blank" href="http://windows.microsoft.com/en-GB/internet-explorer/use-tracking-protection">Microsoft Internet Explorer</a></li>
+                    <li><a target="_blank" href="http://help.opera.com/Windows/12.10/en/notrack.html">Opera</a></li>
+                </ul>
+                <h2>Cookies</h2>
+                <p class="text-center"><strong>You can choose to disable or selectively turn off any cookies or third-party cookies in your browser settings.</strong></p>
+                <p>This site uses cookies for many small incremental improvements. You may find the services function without them, but at a reduced usability. For example, the site will not remember if you have previously paid for elementary OS, and by default you will be asked to pay again.</p>
```
Comment (cassidyjames) [inline privacy-policy.php]: > This site uses cookies for many small incremental improvements. You may find the services function without them, but at a reduced usability. For example, the site will not remember if you have previously paid for elementary OS, and by default you will be asked to pay again.

Change to:

> This site uses cookies for incremental improvements. You may find the services function without them but at a reduced usability. For example, the site will not remember if you have previously paid for elementary OS; by default you will be asked to pay again.

File: privacy-policy.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                <p><strong>CloudFlare</strong> stores cookie to log behavioral elements to analyze for potential threats. The personally identifiable information they store is usually deleted within 4 hours. Cookies enable them to recognize you as a verified human being, and not a robot.</p>
```
Comment (cassidyjames) [inline privacy-policy.php]: > stores cookie to log behavioral elements to analyze for potential threats. The personally identifiable information they store is usually deleted within 4 hours. Cookies enable them to recognize you as a verified human being, and not a robot.

Change to:

> stores cookies to log behavioral elements and analyze potential threats. For more information, see the [CloudFlare Privacy & Security Policy](https://www.cloudflare.com/security-policy).

File: privacy-policy.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                <p><strong>Google Universal Analytics</strong> stores cookies are used to collect information about how visitors use this website, which are then utilized to help improve it. The cookies collect information in an anonymous form, including the number of visitors to the site, where visitors have come to the site from and the pages they visited.</p>
```
Comment (cassidyjames) [inline privacy-policy.php]: > <strong>Google Universal Analytics</strong> stores cookies are used to collect information about how visitors use this website, which are then utilized to help improve it. The cookies collect information in an anonymous form, including the number of visitors to the site, where visitors have come to the site from and the pages they visited.

Change to:

> <strong>Google Analytics</strong> stores cookies to collect information&mdash;including the number of visitors to this site, from where they were referred, and the pages they visit&mdash;in an anonymous form. This information is used to help improve the site. For more information, see [Safeguarding Your Data](https://support.google.com/analytics/answer/6004245) on Google's site.

File: privacy-policy.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                <p><strong>Stripe</strong> uses cookies to remember your last order, or the country you're from so it knows what card types to offer for payments.</p>
```
Comment (cassidyjames) [inline privacy-policy.php]: > <strong>Stripe</strong> uses cookies to remember your last order, or the country you're from so it knows what card types to offer for payments.

Change to:

> <strong>Stripe</strong> uses cookies to remember your last order and your country so it knows what card types to offer for payments.

File: privacy-policy.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+                <p>Our third party services privacy policies can viewed at the links below.</p>
+                <ul>
+                    <li><a target="_blank" href="https://www.cloudflare.com/security-policy">CloudFlare (Distribution and Security)</a></li>
+                    <li><a target="_blank" href="http://www.google.co.uk/intl/en/analytics/learn/privacy.html">Google Universal Analytics</a></li>
+                    <li><a target="_blank" href="https://stripe.com/privacy">Stripe (Payment Processor)</a></li>
+                </ul>
```
Comment (cassidyjames) [inline privacy-policy.php]: This section can be removed since we're adding their links more contextually.

File: privacy-policy.php [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                <h1>Privacy Policy</h1>
+                <h2>Do Not Track</h2>
+                <p>This site honors the do-not-track setting found in most modern browsers, and disables Google Universal Analytics when it is present. Unfortunately, we cannot remove other tracking methods without severely impacting the use of the site. Should you wish to do so, reference the section on disabling cookies.</p>
+                <p>Below you will find links to enabling DNT in your browser.</p>
+                <ul>
+                    <li><a target="_blank" href="https://support.google.com/chrome/answer/114836">Google Chrome</a></li>
+                    <li><a target="_blank" href="https://www.mozilla.org/en-US/firefox/dnt/">Mozilla Firefox</a></li>
+                    <li><a target="_blank" href="http://windows.microsoft.com/en-GB/internet-explorer/use-tracking-protection">Microsoft Internet Explorer</a></li>
+                    <li><a target="_blank" href="http://help.opera.com/Windows/12.10/en/notrack.html">Opera</a></li>
+                </ul>
```
Comment (cassidyjames) [inline privacy-policy.php]: I really don't think we need to include the above section. If they know what DNT is or already have it enabled, fine. But it's not our job to educate people on how to enable it.

**Comment by cassidyjames** [comment]
Okay, the comments look weird on the Pull Request overview, but make more sense if you view them inline in the code changes.

**Comment by danrabbit** [comment]
Can we get line breaks after each sub section header in "Cookies"?

- [ ] (After Cassidy's inline link changes) For external links at the end of a paragraph I think it makes sense to use the "read-more" class.

**Comment by lewisgoddard** [comment]
What do you mean by sub-section headers?
The product names in bold?

**Pull request merged**